### ISSUE TITLE
Raleway "Normal" is broken on Google Docs

### ISSUE THREAD

**Issue by juhani** [impallari/Raleway#29]
This happened at some point during the past year - on Google Docs, the "Normal" variant on Raleway is really bold, even bolder than the normal 'Bold'. I have reported this multiple times to Google, but have not heard back. 

This is really annoying problem, as we used to have Raleway as the default font in all our company templates. So all body text, which used to be slim, turned really bold. 

Can anyone help?

<img width="504" alt="screen shot 2018-07-04 at 14 29 54" src="https://user-images.githubusercontent.com/320317/42274906-129e2c54-7f97-11e8-9fc7-177ab9739393.png">

**Comment by davelab6** [comment]
Does uninstalling the fonts help?

**Comment by jhoang123** [comment]
Yeah I had to remove the previous version of Raleway and install version v4020. A restart is required to fully remove the old font and in some of the cases to get v4020 to work.

https://github.com/impallari/Raleway

https://github.com/impallari/Raleway/issues/27

**Comment by juhani** [comment]
I removed and closed Google Docs, opened again - but it does not help. It shows the broken (bold) version already when adding the font: 

<img width="904" alt="screen shot 2018-07-05 at 17 01 55" src="https://user-images.githubusercontent.com/320317/42327889-9343eb6c-8075-11e8-87ad-bee504d25e64.png">
